jetty-integrations/jetty-infinispan/jetty-infinispan-common/src/main/java/org/eclipse/jetty/session/infinispan/InfinispanSessionData.java

```diff
@@ -32,6 +33,7 @@
  * pool and thus these threads have no knowledge of the correct classloader to
  * use.
  */
+@Indexed
```

Comment:
I'm not sure this is going to work - the `expiry` field which should be indexed can't have an annotation on it, (because it is in the `SessionData` class) and it looks like in the infinispan doc it should have an annotation?

Reply:
oops forgot this. Good catch!
done by adding 
```
    @Override
    @GenericField(searchable = Searchable.YES)
    public long getExpiry() {
        return super.getExpiry();
    }
```